.github/workflows/codeql.yml

```diff
@@ -11,9 +13,6 @@ on:
       - '**/tests/integration/*'
   schedule:
     - cron: "1 18 * * 0"
-  merge_group:
-    types: [checks_requested]
-    branches: [main]
```

Comment:
Can you expand on why this change is necessary? Why should this run on push instead of merge_group?

Reply:
Initially, these checks wouldn't start for [#4652](https://github.com/aws/s2n-tls/pull/4652), but upon review, they are _now_ triggering as expected.  I can't find a github status error explaining why, but this PR does now appear unnecessary.